staging_service/autodetect/Mappings.py

```diff
@@ -55,11 +55,11 @@
 def _flatten(some_list):
     return list(itertools.chain.from_iterable(some_list))
 
-_GZIP_EXT = ["", ".gz", ".gzip"]  # empty string to keep the uncompressed extension
+_COMP_EXT = ["", ".gz", ".gzip"]  # empty string to keep the uncompressed extension
```

Comment:
_COMPRESSION_EXT might be more clear

Reply:
I was thinking the same thing but I didn't want to make the lines too long, but since you think so as well changed